runtimes/node-17.0/server.js

```diff
@@ -38,8 +43,10 @@ const server = micro(async (req, res) => {
             await userFunction(request, response);
         }
     } catch (e) {
-        send(res, 500, e.code === 'MODULE_NOT_FOUND' ? "Code file not found." : e.stack || e);
+        send(res, 500, {stdout: logs, stderr: e.code === 'MODULE_NOT_FOUND' ? "Code file not found." : e.stack || e});
     }
+    logs.length = 0;
+    console.log = console.stdlog;
 });
```

Comment:
This sounds unsafe. Why not `logs = []`, and why are we changing value of `console.log` but not `console.error` and others?

Reply:
what should I capture, let me know, I just used the StackOverflow reference you provided me and implemented this.

Reply:
![CleanShot 2022-04-27 at 09 19 05](https://user-images.githubusercontent.com/19310830/165463344-d64eca5a-ecaf-4276-8efc-1670e616c3fa.png)

There are a few of them: `debug()`, `error()`, `info()`, `log()`, `warn()`. We just need to capture all of them, the same way we do it for `log()`.